scripts/integration/datadog-agent/test.yaml

```diff
@@ -11,7 +11,7 @@ env:
   TEST_DATADOG_API_KEY:
 
 matrix:
-  version: ['7']
+  version: ['7.54']
```

Comment:
Did this turn out to be necessary?

Reply:
Actually looks like `7` will resolve to 7.55 which should be fine.